### ISSUE TITLE
Bug: Review no-member error for Exception status_code member

### ISSUE THREAD

**Issue by rnowling-memphis** [memphisdev/memphis.py#159]
Running Pylint on master gives the following error:

```
memphis/producer.py:210:45: E1101: Instance of 'Exception' has no 'status_code' member (no-member)
```

We should fix this or disable it on a per-line basis using:

```python
# pylint: disable-next=no-member
print("A no-member error will be suppressed for this line
```

**Comment by Adarsh-jaiss** [comment]
will this code snippet work?

```
if self.connection.schema_updates_data[self.internal_station_name] != {}:
                
            # pylint: disable-next=no-member
            print("A no-member error will be suppressed for this line
            msg_to_send = ""
            if hasattr(message, "SerializeToString"):
                msg_to_send = message.SerializeToString().decode("utf-8")
            elif isinstance(message, bytearray):
                msg_to_send = str(message, "utf-8")
            else:
                msg_to_send = str(message)
```

**Comment by rnowling-memphis** [comment]
Thanks for looking into this, @Adarsh-jaiss !  I think the line numbers have shifted since I filed the issue.  I believe Pylint is referring to this line: https://github.com/memphisdev/memphis.py/blob/master/memphis/producer.py#L269 .

Would you mind checking out the repo and running Pylint on your local machine to verify the line number?  Thanks!

**Comment by Adarsh-jaiss** [comment]
@rnowling-memphis I have tried running the pylint on my local machine and fixed this error :

`memphis/producer.py:210:45: E1101: Instance of 'Exception' has no 'status_code' member (no-member)`

and after fixing that, this error occured on the same line  `269:12`: 
`R1720: Unnecessary "else" after "raise", remove the "else" and de-indent the code inside it (no-else-raise)` 

and then I also fixed that and here's the final result : 
```
if hasattr(e, "status_code") and e.status_code == "503":
    raise MemphisError(
        "Produce operation has failed, please check whether Station/Producer still exist"
    )
raise MemphisError(str(e)) from e


```

Is it okay? 
Should i make the PR?

**Comment by Adarsh-jaiss** [comment]
@rnowling-memphis I have created a PR, kindly review it!

Thank you so much for the guiding me ❤️

**Comment by rnowling-memphis** [comment]
This was resolved by #178
